application/src/main/java/uk/nhs/hee/tis/revalidation/controller/DoctorsForDBController.java

```diff
@@ -160,13 +160,25 @@ public ResponseEntity<TraineeSummaryDto> getDoctors(
       @PathVariable(required = false) final List<String> gmcIds) {
     log.info("Receive request to get designatedBodyCode for user: {}", gmcIds);
     if (Objects.nonNull(gmcIds)) {
-      final var doctors = doctorsForDBService
-          .getDoctorsByGmcIds(gmcIds);
+      final var doctors = doctorsForDBService.getDoctorsByGmcIds(gmcIds);
       return ResponseEntity.ok().body(doctors);
     }
     return ResponseEntity.ok().body(TraineeSummaryDto.builder().build());
   }
 
+  @ApiOperation(value = "Get autocomplete value for field", notes = "It will return matching field values", response = ResponseEntity.class)
+  @ApiResponses(value = {
+      @ApiResponse(code = 200, message = "Suggested field values", response = ResponseEntity.class)})
+  @GetMapping("/autocomplete")
+  public ResponseEntity<List<String>> getAutocompleteValesForField(
+      @NonNull @RequestParam(name = AUTOCOMPLETE_FIELD) final String fieldName,
+      @RequestParam(name = INPUT) final String input,
```

Comment:
Did this need to be defined as `requied = false`?

Reply:
makes sense to me